config/vufind/searches.ini

```diff
@@ -254,10 +254,13 @@ WorkKeys = year
 ;
 ; Available modules recommended for use in the side area:
 ;
-; CatalogResults:[GET parameter]:[result limit]
+; CatalogResults:[GET parameter]:[result limit]:[ini section]:[custom heading]
 ;       Display catalog search results matching the terms found in the specified
 ;       GET parameter (default = "lookfor"), limited to a specified number of
-;       matches (default = 5).  This is designed for use with non-catalog modules.
+;       matches (default = 5). This is designed for use with non-catalog modules.
+;       Hidden filters may be defined in the specified section of the searches.ini
+;       file. The optional custom heading overrides the default for this
```

Comment:
I would add "See [CatalogResultsVideoFilter] below for an example of this."

Reply:
Done.